src/routes/api/users.js

```diff
@@ -26,4 +22,5 @@ router.post("/user", (req, res, next) => {
   });
 });
 
+router.post("/auth/siginIn",validUser,login);
 export default router;
```

Comment:
Line 6 seems to be a duplicate of line 25. It's better you remove the nonfunctioning one.